auth-api/src/auth_api/exceptions/errors.py

```diff
@@ -23,6 +23,7 @@
 class Error(Enum):
     """Error Codes."""
 
+    INVALID_ORG = "The organization ID is in an incorrect format.", HTTPStatus.BAD_REQUEST
```

Comment:
Added this consolidate duplicate code in the org_products route for validating orgId format